data/command.go

```
@@ -5,47 +5,75 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"strconv"
 	"strings"
 
 	"github.com/privatix/dappctrl/data/migration"
 	"github.com/privatix/dappctrl/statik"
 	reform "gopkg.in/reform.v1"
 )
 
+type cmdFlag int
+
+// cmdFlags enum
+const (
+	ConnectionString cmdFlag = iota
+	Version
+)
+
 // ExecuteCommand executes commands to manage database
-// db-migrate - command for execute migration scripts
-// db-init-data - command for initialize database by default values.
+// db-migrate - command to execute migration scripts
+// db-init-data - command to initialize database by default values
+// db-version - command to print the version of the database schema.
 func ExecuteCommand(args []string) error {
 	if len(args) == 0 {
 		return nil
 	}
 
 	switch args[0] {
 	case "db-migrate":
-		connStr := getConnectionString(args)
-		if err := migration.Update(connStr); err != nil {
+		m := readFlags(args)
+		connStr := m[ConnectionString]
+		version, _ := strconv.ParseInt(m[Version], 10, 64)
```

lets not ignore the error.

---

updated